src/Entity/LokaalPersoon.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\LokaalPersoonRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=LokaalPersoonRepository::class)
+ */
+class LokaalPersoon
```

Comment:
Since external users can't or shouldn't be included in the database, and internal users are stored in the OIDC store, this class should be removed and later implemented through fetching over an API